Gordon360/ApiControllers/VictoryPromiseController.cs

```diff
@@ -44,13 +44,13 @@ public IHttpActionResult Get()
             var username = authenticatedUser.Claims.FirstOrDefault(x => x.Type == "user_name").Value;
 
             var id = _accountService.GetAccountByUsername(username).GordonID;
-
+            //var id = _accountService.GetAccountByUsername(username).GordonID;
             var result = _victoryPromiseService.GetVPScores(id);
                 if (result == null)
```

Comment:
This indentation looks odd.  It was already this way (you didn't do it), but it would be nice to fix while you're here.